qiita_pet/templates/study_description_templates/raw_data_editor_tab.html

```diff
@@ -12,10 +12,12 @@
         <ol>
           <!-- Step 1: choose the prep template file -->
           <li>
-            select your prep template file
+            select your prep template file (only files with the "txt" and "tsv" file extensions will be displayed here):
```

Comment:
Can you do a `<br />` before the added text to break it up a bit? It's really long right now. Same for below.